tape/src/main/java/com/squareup/tape2/TapeException.java

```diff
@@ -0,0 +1,12 @@
+package com.squareup.tape2;
+
+/** Thrown when an internal error has occurred in tape lib **/
+public class TapeException extends RuntimeException {
+
+    private static final long serialVersionUID = 5138225684096988531L;
```

Comment:
Throwable implements Serializable.
That being said, I'd prefer not to empower such usage.